config/confighttp/README.md

```diff
@@ -87,4 +90,10 @@ receivers:
             - Example-Header
           max_age: 7200
         endpoint: 0.0.0.0:55690
-```
+processors:
```

Comment:
You also need to enable the "include_metadata", no?

Reply:
Sorry, I forget it.  need to enable the "include_metadata"